src/core/java/com/enderio/core/client/rendering/Animation.java

```diff
@@ -0,0 +1,85 @@
+package com.enderio.core.client.rendering;
+
+import net.minecraft.util.Mth;
+
+// TODO: Support more interpolation methods, if required.
+/**
+ * Class for animating between two values over an amount of time, using
+ * {@link Mth.lerp}.
```

Comment:
Fix link using # and put in one line
```suggestion
 * Class for animating between two values over an amount of time, using {@link Mth#lerp}.
```